src/components/containers/menu/Menu.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ToggleButton from '../../controls/buttons/ToggleButton';
+import styled, { ThemeProvider } from 'styled-components';
 import MenuPanel from './MenuPanel';
 import MenuSection from './MenuSection';
+import defaultTheme from '../../theme/defaultTheme';
+import { buttonStyleTypes } from '../../controls/buttons/button-variants';
+
+const StyledToggleButton = styled(ToggleButton)``;
```

Comment:
I don't think these need to be styled components if there are no styles being attached to them.